packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -10,7 +10,7 @@
   "force_source_push_default_scratch_org_text": "SFDX: Push Source to Default Scratch Org",
   "force_source_push_force_default_scratch_org_text": "SFDX: Push Source to Default Scratch Org and Override Conflicts",
   "force_source_status_text": "SFDX: View All Changes (Local and in Default Scratch Org)",
-  "force_apex_test_run_text": "SFDX: Invoke Apex Tests...",
+  "force_apex_test_run_text": "SFDX: Run Apex Tests...",
```

Comment:
@hbc-bindu should the command text include the ellipses? or do we prefer it without? it doesn't look like the lightning run test command includes it

Reply:
Yes @AnanyaJha, we don't need ellipsis in the command text.

Reply:
Changes Done. Please look over it.